### PR TITLE
[DOCS-5536] Updates to Custom Code Vulnerability for beta

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do?
This changes are related to the public beta of custom code vulnerability detection capability. This capability is part of the Vulnerability Management product.

This public beta is scheduled to launch the week of May 29.

### Motivation
[DOCS-5536](https://datadoghq.atlassian.net/browse/DOCS-5536)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5536]: https://datadoghq.atlassian.net/browse/DOCS-5536?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/security/application_security/enabling/java.md [hunk 1]
```diff
@@ -118,9 +118,96 @@ java -javaagent:dd-java-agent.jar \
...
+
+From the command line:
+
+ ```shell
+   java -javaagent:/path/to/dd-java-agent.jar -Ddd.iast.enabled=true -Ddd.service=<MY SERVICE> -Ddd.env=<MY_ENV> -jar path/to/app.jar
+   ```
+
+Or one of the following methods, depending on where your application runs:
+
+**Note**: Read-only file systems are not currently supported. The application must have access to a writable `/tmp` directory.
```
Comment (kayayarai) [inline content/en/security/application_security/enabling/java.md:139]:
```suggestion
**Note**: Read-only file systems are not supported. The application must have access to a writable `/tmp` directory.
```

File: content/en/security/application_security/enabling/nodejs.md [hunk 2]
```diff
@@ -151,9 +151,93 @@ DD_APPSEC_ENABLED=true node app.js
...
+
+To leveraging custom code vulnerability detection capabilities for your service:
+
+1. [Update your Datadog Agent][4] to at least version 7.41.1.
+2. Update your tracing library to at least the minimum version needed to turn on custom code vulnerability detection. For details, see [ASM capabilities support][3].
+3. Add the `DD_IAST_ENABLED=true` environment variable to your application configuration.
+
+If you initialize the APM library on the command line using the `--require` option to Node.js:
+
+ ```shell
```
Comment (kayayarai) [inline content/en/security/application_security/enabling/nodejs.md:166]:
~~~suggestion
   If you initialize the APM library on the command line using the `--require` option to Node.js:

   ```shell
~~~

File: content/en/security/application_security/enabling/java.md [hunk 3]
```diff
@@ -118,9 +118,96 @@ java -javaagent:dd-java-agent.jar \
...
+  }
+]
+```
+
+{{% /tab %}}
+
+   {{< /tabs >}}
+
+4. Restart your service.
+5. To see Application Vulnerability Management for custom code vulnerabilities in action, browse your service and the custom code vulnerabilities will appear in the [Vulnerability Explorer][4]. The `SOURCE` column shows the Custom Code value.
```
Comment (kayayarai) [inline content/en/security/application_security/enabling/java.md:198]:
```suggestion
5. To see Application Vulnerability Management for custom code vulnerabilities in action, browse your service and the custom code vulnerabilities appear in the [Vulnerability Explorer][4]. The `SOURCE` column shows the Custom Code value.
```

File: content/en/security/application_security/enabling/java.md [hunk 4]
```diff
@@ -118,9 +118,96 @@ java -javaagent:dd-java-agent.jar \
...
+
+To leveraging custom code vulnerability detection capabilities for your service:
+
+1. [Update your Datadog Agent][6] to at least version 7.41.1.
+2. Update your tracing library to at least the minimum version needed to turn on custom code vulnerability detection. For details, see [ASM capabilities support][3].
+3. Add the `DD_IAST_ENABLED=true` environment variable to your application configuration.
+
+From the command line:
+
+ ```shell
```
Comment (kayayarai) [inline content/en/security/application_security/enabling/java.md:133]:
~~~suggestion
   From the command line:

   ```shell
~~~

File: content/en/security/application_security/enabling/java.md [hunk 5]
```diff
@@ -118,9 +118,96 @@ java -javaagent:dd-java-agent.jar \
 
 {{< img src="/security/application_security/appsec-getstarted-threat-and-vuln.mp4" alt="Video showing Signals explorer and details, and Vulnerabilities explorer and details." video="true" >}}
 
+## Enabling custom code vulnerability detection
+
+If your service is running a [tracing library version that supports Vulnerability Management for Custom Code Vulnerability Detection][3], enable it by setting the `DD_IAST_ENABLED=true` environment variable and restart your service.
```
Comment (kayayarai) [inline content/en/security/application_security/enabling/java.md:123]:
```suggestion
If your service runs a [tracing library version that supports Vulnerability Management for Custom Code Vulnerability Detection][3], enable the capability by setting the `DD_IAST_ENABLED=true` environment variable and restarting your service.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/security/application_security/enabling/nodejs.md [hunk 6]
```diff
@@ -151,9 +151,93 @@ DD_APPSEC_ENABLED=true node app.js
...
+    "value": "true"
+  }
+]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+4. Restart your service.
+5. To see Application Vulnerability Management for custom code vulnerabilities in action, browse your service and the custom code vulnerabilities will appear in the [Vulnerability Explorer][5]. The `SOURCE` column shows the Custom Code value.
```
Comment (kayayarai) [inline content/en/security/application_security/enabling/nodejs.md:228]:
```suggestion
5. To see Application Vulnerability Management for custom code vulnerabilities in action, browse your service and the custom code vulnerabilities appear in the [Vulnerability Explorer][5]. The `SOURCE` column shows the Custom Code value.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/security/application_security/enabling/java.md [hunk 7]
```diff
@@ -118,9 +118,96 @@ java -javaagent:dd-java-agent.jar \
...
+
+   From the command line:
+
+   ```shell
+   java -javaagent:/path/to/dd-java-agent.jar -Ddd.iast.enabled=true -Ddd.service=<MY SERVICE> -Ddd.env=<MY_ENV> -jar path/to/app.jar
+   ```
+
+Or one of the following methods, depending on where your application runs:
+
+**Note**: Read-only file systems are not supported. The application must have access to a writable `/tmp` directory.
```
Comment (kayayarai) [inline content/en/security/application_security/enabling/java.md:139]:
```suggestion
   Or one of the following methods, depending on where your application runs:

   **Note**: Read-only file systems are not supported. The application must have access to a writable `/tmp` directory.
```

**Review by kayayarai — APPROVED** [review]

**Review by gorkavicente — CHANGES_REQUESTED** [review]
/security/application_security/enabling/compatibility/
Replace Java version with 1.15.0

/security/application_security/enabling/compatibility/java/
Vulnerability Management
–> Custom code vulnerability detection 1.15.0

The minimum tracer version to get all supported ASM capabilities for Java is 1.15.0.

**Pull request merged**